tests/src/Kernel/fixtures/rendering.yml

```diff
@@ -440,6 +440,59 @@
       'div.ecl-container div.ecl-banner__container div.ecl-banner__content div.ecl-banner__title span.ecl-banner__title-text': Banner title
       'div.ecl-container div.ecl-banner__container div.ecl-banner__content p.ecl-banner__description span.ecl-banner__description-text': Banner description
       'a.ecl-link.ecl-link--cta.ecl-link--icon.ecl-banner__link-cta[href="http://example.com"] span.ecl-link__label': Subscribe
+- array:
+    '#type': pattern
+    '#id': banner
+    '#fields':
+      title: "Banner video title"
+      url: "http://example.com"
+      label: "Subscribe"
+      size: "l"
+      font_size: "l"
+      font_color: "dark"
+      box_background: "light"
+      description: "Banner video description"
+      horizontal: "center"
+      vertical: "bottom"
+      full_width: true
+      credit: "© Copyright or credit"
+      link_extra_attributes:
+        - name: 'data-name'
+          value: 'data value'
+      video:
+        poster: "https://inno-ecl.s3.amazonaws.com/media/examples/example-image.jpg"
+        sources:
+          - src: "https://inno-ecl.s3.amazonaws.com/media/videos/big_buck_bunny.mp4"
+            src_type: "video/mp4"
+        tracks:
+          - src: "captions/bunny-en.vtt"
+            kind: "captions"
+            src_lang: "en"
+            label: "English"
+      sr_play: "Play video"
+      sr_pause: "Pause video"
+  assertions:
+    count:
+      'section.ecl-banner.ecl-banner--font-l.ecl-banner--box-bg-light.ecl-banner--color-dark.ecl-banner--l.ecl-banner--full-width.ecl-banner--h-center.ecl-banner--v-bottom': 1
+      'section.ecl-banner.ecl-banner--font-l.ecl-banner--box-bg-light.ecl-banner--color-dark.ecl-banner--l.ecl-banner--h-center.ecl-banner--v-bottom': 1
+      'div.ecl-banner__content div.ecl-banner__cta a.ecl-link.ecl-link--cta.ecl-link--icon.ecl-banner__link-cta[href="http://example.com"]': 1
+      'div.ecl-banner__content div.ecl-banner__cta a.ecl-link.ecl-link--cta.ecl-link--icon.ecl-banner__link-cta[data-name="data value"]': 1
+      'svg.ecl-icon.ecl-icon--xs.ecl-icon--rotate-90.ecl-link__icon': 1
+      'picture.ecl-picture.ecl-banner__picture img.ecl-banner__image[src="http://via.placeholder.com/150x150"]': 0
+      'video[poster="https://inno-ecl.s3.amazonaws.com/media/examples/example-image.jpg"]': 1
```

Comment:
Let's add the attributes (autoplay, loop and muted) of the video tag in the assertion as they are a requirement for ewcms but ECL adds them now, so in case they remove them we know we have to add them back.